CameraXBasic/app/src/main/java/com/android/example/cameraxbasic/MainActivity.kt

```diff
@@ -75,4 +77,17 @@ class MainActivity : AppCompatActivity() {
                 mediaDir else appContext.filesDir
         }
     }
+
+    private fun hideSystemUI() {
+        WindowCompat.setDecorFitsSystemWindows(window, false)
```

Comment:
Is this a new case we had to handle because of updates to WindowInsets?

Sorry I don't know much about the WindowInsets APIs. Wanted to understand why we need this logic now, or if it's just something the sample was always missing.

Reply:
The old ways triggered warning: asking to change.

Reply:
ah, interesting, okay thanks for clarifying!

Reply:
Looking at the old flags, `setDecorFitsSystemWindows(..., false)` makes sense here to me! On older APIs, it translates to

```
View.SYSTEM_UI_FLAG_LAYOUT_STABLE
   | View.SYSTEM_UI_FLAG_LAYOUT_HIDE_NAVIGATION
   | View.SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN
```

which the `FLAGS_FULLSCREEN` had (except `SYSTEM_UI_FLAG_LAYOUT_STABLE`)